src.ts/debug.ts

```diff
@@ -59,7 +52,12 @@ export class Debug {
     };
 
     // TODO: Update Service to provide types
-    common.Service.call(this, config, options);
+    super(config, options || {});
```

Comment:
I've updated the code to move the `normalizeArguments` call after the call to `super`.  PTAL